tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import salt.modules.cmdmod as cmdmod
+import salt.utils.path
+
+pytestmark = [
+    pytest.mark.windows_whitelisted
```

Comment:
If you add a trailing comma, black won't reformat this code chunk

Reply:
Done